plugins/fluentd_telemetry_plugin/src/streamer.py

```diff
@@ -46,9 +47,13 @@ class UFMTelemetryStreaming(metaclass=SingletonMeta):
     UFMTelemetryStreaming class
     to manage/control the streaming
     """
+    def __init__(self, conf_parser=None):
```

Comment:
you can remove the conf_parser from the constructor. and import it like this:
```python
self.config_parser = UFMTelemetryStreamingConfigParser()
```

because also UFMTelemetryStreamingConfigParser is a singleton